apps/meteor/app/api/server/v1/misc.ts

```diff
@@ -634,9 +635,8 @@ API.v1.addRoute(
 	{ authRequired: true },
 	{
 		async get() {
-			const isMailURLSet = !(process.env.MAIL_URL === 'undefined' || process.env.MAIL_URL === undefined);
-			const isSMTPConfigured = Boolean(settings.get('SMTP_Host')) || isMailURLSet;
-			return API.v1.success({ isSMTPConfigured });
+			const smtpConfigured = isSMTPConfigured();
+			return API.v1.success({ smtpConfigured });
```

Comment:
This is a smol breaking change. We should keep same return value

Reply:
Thanks for noticing that man!